Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -287,6 +287,14 @@
         "Trytodeploywebappagainwithappofflineoptionselected": "Try to deploy web app again with app_offline option selected.",
         "UpgradeAgent": "Task requires a 2.104.1 agent or higher. Please upgrade the agent",
         "NOJSONfilematchedwithspecificpattern": "NO JSON file matched with specific pattern",
-        "JSONvariablesubstitutioncanonlybeappliedforJSONfiles": "JSON variable substitution can only be applied for JSON files"
+        "JSONvariablesubstitutioncanonlybeappliedforJSONfiles": "JSON variable substitution can only be applied for JSON files",
```

Comment:
As part of @vincentdass PR , changes corresponding to variable substitution in task.json has been added.